rs/nns/governance/src/governance.rs

```diff
@@ -584,160 +583,7 @@ impl NnsFunction {
                 | NnsFunction::RetireReplicaVersion
         )
     }
-}
-
-impl ManageNeuronResponse {
```

Comment:
As mentioned earlier, this was transplanted to rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs (hash tag dff stats inflation).